backend/alerts/integrations/microsoft-teams/templates_test.go

```diff
@@ -0,0 +1,124 @@
+package microsoft_teams
```

Comment:
My treatment of adaptive cards can be error prone. 
Tests need to exist of all templates.